src/workerd/jsg/util.c++

```diff
@@ -806,4 +806,29 @@ v8::MaybeLocal<v8::String> newExternalTwoByteString(Lock& js, kj::ArrayPtr<const
   return ExternTwoByteString::createExtern(js.v8Isolate, buf);
 }
 
+void captureStackTrace(v8::Isolate* isolate, v8::Local<v8::Object> target) {
+  auto context = isolate->GetCurrentContext();
+  auto global = context->Global();
+
+  auto name = v8StrIntern(isolate, "captureStackTrace");
+  // This is a bit of a hacky trick. The context has this additional "extra bindings object"
+  // attached to it that v8 uses to expose some built ins. For instance, it is used by Node.js
+  // to get access to the Trace and IsTraceCategoryEnabled built-in functions. We use it here
+  // to cache the reference to the Error.captureStackTrace function.
+  auto extras = context->GetExtrasBindingObject();
```

Comment:
fwiw, I'm happy *not* caching it like this if folks disagree with this approach.

Reply:
note also that we could fairly easily patch v8 so that it exposes the `Error.captureStackTrace` built-in on the extras binding object from the start.

Reply:
Why cache it on this object and not just in a `V8Ref` from C++?

Reply:
Really just so we don't have to think about where it's held. We could do either. Really tho I'm thinking about a v8 patch that will have v8 just expose the `captureStackTrace` here automatically so we don't have to capture and cache it at all.

Reply:
FWIW, it's like a 1-2 line patch in v8